src/components/VmsPageHeader/style.css

```diff
@@ -0,0 +1,2 @@
+.user-name {
```

Comment:
I know this was probably a blind move of code but maybe can stop and think if we need this file at all :-) What do you think @mareklibra ?